examples/tdigestExample.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"fmt"
+	"github.com/davecgh/go-spew/spew"
+	"github.com/grafadruid/go-druid"
+	"github.com/grafadruid/go-druid/builder"
+	"github.com/grafadruid/go-druid/builder/aggregation"
+	"github.com/grafadruid/go-druid/builder/datasource"
+	"github.com/grafadruid/go-druid/builder/granularity"
+	"github.com/grafadruid/go-druid/builder/intervals"
+	"github.com/grafadruid/go-druid/builder/postaggregation"
+	"github.com/grafadruid/go-druid/builder/query"
+	"log"
+	"time"
+)
+
+func getConnection() *druid.Client {
+	d, err := druid.NewClient("http://localhost:8888")
+	if err != nil {
+		log.Fatal(err)
+	}
+	status, _, err := d.Common().Status()
+	if err != nil {
+		log.Fatal(err)
+	}
+	fmt.Println("{\"version\": \"" + status.Version + "\"}")
+	return d
+}
+
+// tdigestSketchUsingBuilder example using Builder Pattern
+func tdigestSketchUsingBuilder() {
+	d := getConnection()
+	table := datasource.NewTable().SetName("rollup-data")
```

Comment:
could you please add documentation on how to create this data-source?

Reply:
done